crates/forge/bin/cmd/test/mod.rs

```diff
@@ -308,14 +316,25 @@ impl TestArgs {
             .profiles(profiles)
             .build(&output, project_root)?;
 
+        let should_debug = self.debug.is_some();
+
+        if self.flamegraph && self.flamechart {
```

Comment:
This check should be a `#[arg(... conflicts_with = "...")]` in the clap args above